### ISSUE TITLE
Object detection using YOLO-NAS model in C++ or c#

### ISSUE THREAD

**Issue by R-Mahmoudi** [Deci-AI/super-gradients#933]
I am trying to make inferences using the YOLO-NAS model YOLO-NAS from a C# / c++ program. I am aware of the github port, but would like to use Ncnn Ncnn or other framework Tensorflow and PyTorch and Onnx runtime  in c++ or c# .

**Comment by BloodAxe** [comment]
Good luck with that

пт, 5 трав. 2023 р. о 18:02 dagshub[bot] ***@***.***> пише:

**Comment by BloodAxe** [comment]
Closing as it's not directly related to SuperGradients and no clear call to action provided. 
We suggest to follow the documentation of corresponding frameworks.
https://pytorch.org/cppdocs/
https://github.com/leimao/ONNX-Runtime-Inference

**Comment by jason-li-831202** [comment]
I have implemented the C++ version of OnnxRuntime in my project, which might be helpful to you : 
https://github.com/jason-li-831202/YOLO-NAS-onnxruntime
